### ISSUE TITLE
Production use

### ISSUE THREAD

**Issue by mihai1voicescu** [shepeliev/webrtc-kmp#29]
First of all good job with the library! There is a real need for a proper KMP WebRTC lib.

Before committing to this library we had some questions, if you would be so kind as to provide an answer to them:

1. In your personal opinion how production ready is the lib in its current form?
2. What are the plans to support the library in the future?
3. Do you accept PRs if needed?
4. What is the motivation behind creating this open-source library?

**Comment by shepeliev** [comment]
Thanks for the questions.

1. Definitely, It's not production-ready yet. There is a lack of some API (eg. stat reports), also it's poorly tested with iOS and JS. Unfortunately, I can't devote a lot of time at the moment because of my current job. However, I'll be available for full-time development in a couple of months.
2. Actually, I'd like this lib to be used widely in other KMP apps. So yes, I have strong plans for maintaining it in the future.
3. Sure, PRs are welcome :)
4. I have several motivation points. 
- The first one, I have a pet Android app that depends on WebRTC. I'm going to port it to iOS and Web. And I want to do it with KMP.
- I'm interested in Kotlin Multiplatform, so that I've decided to dive into it with real-world lib.
- One more thing. I've been wanted to start some open source project for a while. So I've got a good reason to do that, as there is no any KMP WebRTC implementation yet.

**Comment by tricknology** [comment]
Hello, I was searching the web for a KMP multiplatform library for a side project I'd like to start up and I came across this one.  I have a very strong Android/Java/Kotlin background; I am a novice with the other supported languages, but I'm interested in joining this effort.  May I?

**Comment by mihai1voicescu** [comment]
@tricknology I don't think anybody has ever said no to help on an opensource project 😄

**Comment by shepeliev** [comment]
Yeah, @mihai1voicescu is absolutely right, you shouldn't hesitate to collaborate if you want to. 

Unfortunately, I can't participate as much as I'd like due to the war in Ukraine, however, I'll try to review PRs as soon as possible :)
